### PR TITLE
milestone 3 Array of objects and Static variable 

### PR DESCRIPTION
**Pull request by popeinabar**
added static variable to define a static integer variable totalTickets within the class Ticket and used array of objects to store Ticket objects.

here is the explanation [video](https://www.loom.com/share/a029dab3e87e4442acd468e21c884be2?sid=7f3a2860-383e-4dc7-ad2f-139c2c21da5b)
here is the [video](https://www.loom.com/share/f1b32c58eba94b4cb3228ba56f2ecb72?sid=fff33ac7-9ca9-450c-9c60-75b1ac9df085) for code run

### CODE REVIEW

**Comment by th3c0d3br34ker** [comment]
thanks @popeinabar for your submission. the self-viva is sufficient for `Static Elements` and `Array of Objects`. 

please go ahead and merge this PR.

File: main.cpp [hunk 1]
```diff
@@ -72,28 +73,38 @@ class Ticket {
     }
 
     static int getTotalTickets() {
-       
+        return totalTickets;
         return 0;
```
Comment (th3c0d3br34ker) [inline main.cpp]: why are there two `return` statements?

**Review by th3c0d3br34ker — APPROVED** [review]

File: main.cpp (hunk 1 above)
Comment (popeinabar) [reply main.cpp]: it was a mistake i was testing the code for that purpose there came 2 return i am removing the unnecessary one

Comment (th3c0d3br34ker) [reply main.cpp]: okay, please fix it before merging it,

Comment (popeinabar) [reply main.cpp]: fixed it can i merge it

**Pull request merged**